docs/how-to-guides/replication.md

```diff
@@ -105,10 +105,11 @@ instances:
    REPLICA instance 1 is called REP1, its replication mode is SYNC, and it is
    located at IP address `172.17.0.3.` with port `10000`.
 
-   When the REPLICA instance is running in SYNC mode, the MAIN instance will not
-   commit a transaction until all REPLICA instances running in the SYNC mode
-   confirm they have received the same transaction. SYNC mode prioritizes data
-   consistency but has no tolerance for any network failures.
+   When the REPLICA instance is running in SYNC mode, after committing a transaction
+   the MAIN instance will communicate the changes to all REPLICA instances running 
+   in SYNC mode and wait until it receives a response or that a timeout is reached. <br/>
+   In case of timeout, it will return an error to the user indicating that he should check the 
+   status of the REPLICAS as there might be a network/hardware issue.
```

Comment:
REPLICAs' statuses as there might be network or hardware issues.

Reply:
Done